content/outbreaks/_index.en.md

```diff
@@ -91,120 +91,3 @@ You will not be able to use the same set up of the data dictionaries, but you sh
 
 There is also coding to be able to guide you on conducting simple statistical comparisons between groups (chi square, t-test etc.), as well as the calculation of Odds Ratios (ORs) and Risk Ratios (RR) and running regression models for multivariate analyses.
```

Comment:
@nsbatra - dont know how much time you have. But if you do, and the other stuff is cleared. 
Could you show some example code comparing HEV rdt postives versus negatives. 
Run tab_univariate to produce odds ratios. And maybe example codes comparing groups with chi-squared tests, and potentially also t-tests (though preferably kruskal-wallis tests). Have example code if you need!

Reply:
This is off-topic, so I raised an issue in #19